### DISCUSSION TITLE
logout from wave app on cloud

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1325] [Ideas]
is there an option to logout the user from the app on the H2O AI cloud env? Note: just the app, and not the complete cloud env.

This is what the H2O Cloud team conveyed:
"If the app itself is OIDC enabled, and you want to log the user out of that, then we do not have any control over that - the session is held by waved and we only proxy traffic"

**Comment by mturoci** [comment]
Not sure about this one - cc @lo5.

Waved has `_auth/logout` that should log you out of the current session, but the request has to contain the cookie with current session ID which I am not sure you can access. Definitely poke around.

Edit:
You cannot call the logout endpoint from python (missing cookie), but need to call it from browser (e.g. via `ui.link`).

**Reply by azim-b** [reply]
is this a path, `/_auth/logout`?

**Reply by mturoci** [reply]
Yep, https://github.com/h2oai/wave/blob/cc91339af202c02bb2dd0c4fde3633b6a043a600/server.go#L86
